src/components/shared/PopoverSection/UserSection.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { defineMessages } from 'react-intl';
+
+import { DropdownMenuItem, DropdownMenuSection } from '~shared/DropdownMenu';
+import NavLink from '~shared/NavLink';
+import { CREATE_USER_ROUTE, USER_EDIT_ROUTE } from '~routes/routeConstants';
+import {
+  useAppContext,
+  useColonyContext,
+  useUserAccountRegistered,
+} from '~hooks';
+
+const displayName = 'PopoverSection.UserSection';
+
+const MSG = defineMessages({
+  buttonGetStarted: {
+    id: `${displayName}.buttonGetStarted`,
+    defaultMessage: 'Get started',
+  },
+  myProfile: {
+    id: `${displayName}.myProfile`,
+    defaultMessage: 'My Profile',
+  },
+  settings: {
+    id: `${displayName}.settings`,
+    defaultMessage: 'Settings',
+  },
+  wallet: {
+    id: `${displayName}.wallet`,
+    defaultMessage: 'Wallet',
+  },
+});
+
+const UserSection = () => {
+  const { colony } = useColonyContext();
+  const { user } = useAppContext();
+  const userHasAccountRegistered = useUserAccountRegistered();
+
+  return (
+    <DropdownMenuSection separator>
+      {userHasAccountRegistered ? (
+        <>
+          <DropdownMenuItem>
+            <NavLink
+              to={`/user/${user?.name}`}
+              text={MSG.myProfile}
+              data-test="userProfile"
+            />
+          </DropdownMenuItem>
+          <DropdownMenuItem>
+            <NavLink
+              to={USER_EDIT_ROUTE}
+              text={MSG.settings}
+              data-test="userProfileSettings"
+            />
+          </DropdownMenuItem>
+        </>
+      ) : (
+        <DropdownMenuItem>
+          <NavLink
+            to={{
+              pathname: CREATE_USER_ROUTE,
+              state: colony?.name
+                ? { colonyURL: `/colony/${colony?.name}` }
+                : {},
```

Comment:
Is this still needed? Looking at (Will's User Creation PR)[https://github.com/JoinColony/colonyCDapp/pull/88/], he doesn't seem to use it anywhere.
In the Dapp it was used in `WalletRequiredRoute` which we've abandoned in the CDapp

Reply:
I think you are right, updated 👍